app/controllers/concerns/api/v3/sync_to_user.rb

```diff
@@ -2,33 +2,30 @@ module Api::V3::SyncToUser
   extend ActiveSupport::Concern
 
   included do
-    def region_records
-      model = controller_name.classify.constantize
-      model.syncable_to_region(current_sync_region)
-    end
-
     def current_facility_records
-      region_records
-        .where(patient: prioritized_patients)
+      model_sync_scope
+        .where(patient: current_facility.syncable_patients.pluck(:id))
```

Comment:
Would a couple semantic methods help with readability?

```ruby
def current_facility_patient_ids
  @current_facility_patient_ids ||= current_facility.syncable_patients.pluck(:id)
end

def other_facility_patient_ids
  @other_facility_patient_ids ||= current_sync_region.syncable_patients.pluck(:id) - current_facility_patient_ids
end
```

Reply:
They might, I can play around with this. But our general approach was to reduce the number of methods in this mixin. Every method that we add, is an opportunity for someone to override it in some random place in some controller that inherits from it. The thinner this mixin is in terms of pure method volume, the cleaner the internal API is.

I'll take a shot at this and see if it helps anyway. Will respond back here.